packages/core/src/BlockNoteEditor.ts

```diff
@@ -34,14 +34,49 @@ export type BlockNoteEditorOptions = {
   // TODO: Figure out if enableBlockNoteExtensions/disableHistoryExtension are needed and document them.
   enableBlockNoteExtensions: boolean;
   disableHistoryExtension: boolean;
+  /**
+   * Factories used to create a custom UI for BlockNote
+   */
   uiFactories: UiFactories;
+  /**
+   * TODO: why is this called slashCommands and not slashMenuItems?
+   *
+   * @default defaultSlashMenuItems from `./extensions/SlashMenu`
+   */
   slashCommands: BaseSlashMenuItem[];
+
+  /**
+   * The HTML element that should be used as the parent element for the editor.
+   *
+   * @default: undefined, the editor is not attached to the DOM
+   */
   parentElement: HTMLElement;
+  /**
+   * An object containing attributes that should be added to the editor's HTML element.
+   *
+   * @example { class: "my-editor-class" }
+   */
   editorDOMAttributes: Record<string, string>;
```

Comment:
I'm thinking we might want to separate this into `editorClass` and `editorDOMAttributes` instead of adding the `defaultStyles`, the reason being we want to apply the same styling to the drag preview but probably don't want to set the same attributes since they might break it. That way, we can just have `editorClass: string` and set it to `defaultStyles` if it's undefined. Then the drag preview can also just inherit `editorClass`.

Reply:
So to just remove the default styles, you would do `editorClass: ""` instead of `defaultStyles: false`. Though in reality idk if someone would remove the default styles without adding their own using a custom class anyway.

Reply:
Hmm, that will have the disadvantage that it's impossible to keep the default styles but add a different class (I think that's what you mean in your last comment).

We can still infer the classnames needed for the dragpreview from `editorDOMAttributes` right?

Or maybe the best way to go about this is directly grabbing the class from the editor element in the DraggableBlocksPlugin. That way, we also don't need to pass the class attributes all the way down

Reply:
Yeah that's a good point, I was thinking about what if someone wanted to add a `styles` attribute to the editor (as it would need to be applied to the drag preview too), but my idea doesn't fix that issue either. Getting the editor class from the DOM element should be ok.